src/mip/HighsTableauSeparator.cpp

```diff
@@ -207,8 +206,8 @@ void HighsTableauSeparator::separateLpSolution(HighsLpRelaxation& lpRelaxation,
 
     lpAggregator.getCurrentAggregation(baseRowInds, baseRowVals, false);
 
-    if (baseRowInds.size() - fracvar.row_ep.size() >
-        1000 + 0.1 * mip.numCol()) {
+    if (10 * (baseRowInds.size() - fracvar.row_ep.size()) >
+        10000 + mip.numCol()) {
```

Comment:
/home/jajhall/HiGHS/src/mip/HighsTableauSeparator.cpp:211:59: warning: comparison of integer expressions of different signedness: ‘std::vector<int>::size_type’ {aka ‘long unsigned int’} and ‘int’ [-Wsign-compare]
  211 |     if (10 * (baseRowInds.size() - fracvar.row_ep.size()) >
      |         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
  212 |         10000 + mip.numCol()) {
      |         ~~~~~~~~~~~~~~~~~~~~

Reply:
Sorry, this one escaped me.

    if (10 * (baseRowInds.size() - fracvar.row_ep.size()) >
        10000 + static_cast<size_t>(mip.numCol())) {